deploy/helm/dynatrace-oneagent-operator/templates/Common/clusterrolebinding.yaml

```diff
@@ -0,0 +1,16 @@
+{{- $plaformIsSet := printf "%s" (required "Platform needs to be set to kubernetes or openshift" (include "dynatrace-oneagent-operator.plaformSet" .))}}
```

Comment:
`platformSet` is?

Reply:
openshift/kubernetes

Reply:
This is necessary to have a check before deploying the yaml if the platform is set at all - otherwise the .yaml files in the 'Common' folder would be deployed there even if no platform is set at all

Reply:
Just realized the typo, `plaformSet`.

Anyway, I know there is a variable `platform` already, just wasn't certain of the difference of it compared to `platformSet` but IIUC, `platformSet` is set to true by `_helpers.tpl` when `platform` is set correctly, so it's a helper variable. Fine with me then.